src/main/java/lotto/util/RandomNumbersGenerator.java

```diff
@@ -0,0 +1,36 @@
+package lotto.util;
+
+import static lotto.domain.Lotto.LOTTO_NUMBERS_SIZE;
+import static lotto.domain.Lotto.SAVED_MAX_NUMBER;
+import static lotto.domain.Lotto.SAVED_MIN_NUMBER;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+import lotto.domain.Number;
+
+public class RandomNumbersGenerator {
+
+    private static final int GENERATE_MIN_NUMBER = 0;
+
+    private static final List<Number> lottoNumbers = IntStream.rangeClosed(SAVED_MIN_NUMBER, SAVED_MAX_NUMBER)
+        .boxed()
+        .map(Number::new)
+        .collect(Collectors.toList());
+
+    private RandomNumbersGenerator() {
+    }
+
+    public static List<Number> generateNumbers() {
+        Collections.shuffle(lottoNumbers);
+        List<Number> numbers = new ArrayList<>();
+        for (int i = GENERATE_MIN_NUMBER; i < LOTTO_NUMBERS_SIZE; i++) {
+            numbers.add(lottoNumbers.get(i));
+        }
+        Collections.sort(numbers);
+        return numbers;
```

Comment:
이렇게 표현 해 볼 수도 있을 것 같습니다 😄 
이 부분은 참고만 부탁 드리겠습니다 🙇 

```suggestion
        return lottoNumbers.stream()
                .limit(LOTTO_NUMBERS_SIZE)
                .sorted()
                .collect(Collectors.toList());
```

Reply:
stream 사용법을 하나 더 알아갑니다 :)